packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -68,6 +69,49 @@ export interface DeploymentCircuitBreaker {
   readonly rollback?: boolean;
 }
 
+/**
+ * The ecs metric name
+ */
+export enum EcsMetric {
```

Comment:
What units are these metrics in? are they percentages?

Reply:
yes the units are in form of percentages
I mentioned this in the description as shown below
`/**
 * The ecs metric name
 * Units of this metrics are in form of percentages
 */
export enum EcsMetric {`